_includes/v22.1/sidebar-data/manage.json

```diff
@@ -205,6 +205,35 @@
         {
           "title": "Secure CockroachDB Self-Hosted Clusters",
           "items": [
+            {
```

Comment:
@MichaelTrestman  reminder that the verb forms should be consistent and the style guide for [tasks](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#task) recommends imperative verb form. Create is imperative, but Managing -> Manage, Using -> Use.  Also, [titles](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#page-title) should be title case, not sentence case.

Reply:
Thanks for the guidance! This is still rough and not really ready for review, I have set it as a 'draft' now.